src/education.js

```diff
@@ -1,13 +1,29 @@
+//ul tag
+let eduTag = document.createElement("ul");
+eduTag.id = "edu-list";
+eduTag.className = "edu-list";
+
+//append ul tag to div
+let eduDiv = document.getElementById("edu-section");
+eduDiv.appendChild(eduTag).innerHTML;
+
+//li tag
+let eduList = document.createElement("li");
+eduList.id = "edu-items";
+eduTag.appendChild(eduList);
+
 function processData(response) {
-    const education = response.education;
-    let output = "";
-    document.getElementById("title").innerHTML = "EDUCATION";
-    for (let i = 0; i < education.length; i++) {
-      output += education[i].degree + "<br>";
-      output += education[i].school + "<br>";
-      output += education[i].date + "<p>";
-    }
-    document.getElementById("list").innerHTML = output;
+  let listArray = document.getElementById("edu-list");
+  document.getElementById("edu-title").innerHTML = "EDUCATION";
+  for (let i = 0; i < response.length; i++) {
+    listArray.appendChild(eduList).innerHTML +=
+      response[i].degree +
+      "<br>" +
+      response[i].school +
+      "<br>" +
+      response[i].date +
+      "<br> <br>";
```

Comment:
The proper way to do this is to create three different `li` elements instead of using line breaks.
The double line break at the end should be a margin-top property on the ul.
Each education item needs to be its own `ul`. Your page should end looking like:
`ul[style=margin-top: 15px]`
`  li.degree ...`
`  li.school Grand Circus`
`  li.date ...`
`/ul`
`ul[style=margin-top: 15px]`
`  li.degree ...`
`  li.school MSU`
`  li.date ...`
`/ul`